src/lib.rs

```diff
@@ -263,6 +264,7 @@ pub use thread_local::{ThreadRng, thread_rng, set_thread_rng, set_new_thread_rng
         random, random_with};
 
 use prng::IsaacWordRng;
+use distributions::range::Range;
```

Comment:
Note: there is a second import around line 472: `use distributions::range::SampleRange;`.
I left it there to avoid a rebase.